spec/integration/support/inspect_unmanaged_files_examples.rb

```diff
@@ -63,6 +63,7 @@ def parse_md5sums(output)
 
       expected = <<EOF
 Inspecting unmanaged-files...
+Note: Using traditional inspection because there is no helper binary for architecture 'x0_0' available.
```

Comment:
I played around with the regex for a while but it became ugly to a degree where I think it's not worth it anymore. Are you fine with leaving it like it is?